src/pages/Home.js

```diff
@@ -4,8 +4,19 @@ const Home = () => {
   return (
 
     <div>
-      <h1>Home </h1>
-      <ul>villains</ul>
+      <h1>Meet the Villains!</h1>
+      <br>
+
+      </br>
+      <p>Sinister Bachelors & Bachelorettes</p>
+
+      <p> What kind of villain are you most attracted to?
+{/*         
+          <img alt= "Devil Emoji" 
+          src="https://cdn.pixabay.com/photo/2020/12/24/01/14/smile-5856241_1280.png"> </img> */}
```

Comment:
Do you need this image?  If so, please comment it back in, if not, please remove it.

Reply:
Plan to use this image.

Reply:
Best practice is to leave main clean of commented-out code unless it is for documentation.  If you plan to use it, it is best to either implement it now by uncommenting or adding it when you plan on using it.